wordpress-mobile/WordPress-iOS
# Update Reader Card Design

**Fixes** #14005

**Subtasks:**
- [X] Relative Date Formatter: https://github.com/wordpress-mobile/WordPress-iOS/pull/14186
- [x] Update cross post design: https://github.com/wordpress-mobile/WordPress-iOS/pull/14207

This applies the new reader card design in both dark and light modes. 

## Screenshots
### Light Mode
| Before | After |
|---|---|
|  ![before-discover-light](https://user-images.githubusercontent.com/793774/82853573-fcc38880-9eba-11ea-8b8c-a3e23a2cc1c9.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-05-25 at 19 36 39](https://user-images.githubusercontent.com/793774/82855060-18c92900-9ebf-11ea-97a2-5663c8668d97.png) |

### Dark Mode
| Before | After |
|---|---|
| ![before-discover-dark](https://user-images.githubusercontent.com/793774/82853616-11078580-9ebb-11ea-88b3-3db125e6ff66.png)|![Simulator Screen Shot - iPhone 11 Pro - 2020-05-25 at 19 36 19](https://user-images.githubusercontent.com/793774/82855087-2c748f80-9ebf-11ea-960b-253eb8398354.png)|

### Other Card States

| Default | Text Only | Photo and title only | 
|---|---|---|
| <img width="377" alt="Screen Shot 2020-05-25 at 7 38 41 PM" src="https://user-images.githubusercontent.com/793774/82855162-69408680-9ebf-11ea-90c0-c3f2aa144eaf.png">|<img width="376" alt="Screen Shot 2020-05-25 at 7 43 14 PM" src="https://user-images.githubusercontent.com/793774/82855381-013e7000-9ec0-11ea-96e5-4ceaa6f82fc6.png"> |<img width="375" alt="Screen Shot 2020-05-25 at 7 44 58 PM" src="https://user-images.githubusercontent.com/793774/82855461-3e0a6700-9ec0-11ea-874a-a843a46e45ec.png">|

| Photo Only | No site icon w/ comments disabled | Title only |
|---|---|---|
|<img width="375" alt="Screen Shot 2020-05-25 at 7 46 18 PM" src="https://user-images.githubusercontent.com/793774/82855545-70b45f80-9ec0-11ea-8677-0b02807b70c6.png">|<img width="375" alt="Screen Shot 2020-05-25 at 7 46 48 PM" src="https://user-images.githubusercontent.com/793774/82855571-8164d580-9ec0-11ea-84c8-72597cfdf75b.png">|<img width="376" alt="Screen Shot 2020-05-25 at 7 41 56 PM" src="https://user-images.githubusercontent.com/793774/82855342-e66bfb80-9ebf-11ea-9b85-007e7331ea94.png">|

### Other Devices
| iPhone 5s | iPad |
|---|---|
| ![Simulator Screen Shot - iPhone 5s - 2020-05-25 at 20 07 13](https://user-images.githubusercontent.com/793774/82856742-b7f01f80-9ec3-11ea-9b11-61a920acdcd3.png) | ![Simulator Screen Shot - iPad Air (3rd generation) - 2020-05-25 at 20 11 02](https://user-images.githubusercontent.com/793774/82856824-fa196100-9ec3-11ea-8e77-d18482711ab5.png)|

### Notes
- This does not include the updated relative time logic yet
- The action icons on the discover view are disabled even though when you tap on the post itself its enabled. This is because the discover post itself has comments disabled, but when you tap on it, it loads the attributed post.
- Some elements have not been updated such as the attribution label since the design wasn't updated for them yet.

## To test:
1. Launch the app
2. Tap on the Reader tab
3. Tap on any section
4. Scroll and make sure the card content and design matches
5. Close the app
6. Enter dark mode
7. Follow steps 1 - 4 to make sure the card looks good in dark mode

## PR submission checklist:
